connection.js

```diff
@@ -1182,23 +1182,23 @@ class Connection {
         const src_port = match[4];
         const dst_port = match[5];
 
+        // TODO: is it at all possible to avoid the ESLint disable here?
         // Validate source/destination IP
         /*eslint no-fallthrough: 0 */
         switch (proto) {
             case 'TCP4':
                 if (ipaddr.IPv4.isValid(src_ip) && ipaddr.IPv4.isValid(dst_ip)) {
-                    break;
+                    break; // Valid IPv4 address: break loop
                 }
             case 'TCP6':
                 if (ipaddr.IPv6.isValid(src_ip) && ipaddr.IPv6.isValid(dst_ip)) {
-                    break;
+                    break; // Valid IPv6 address: break loop
                 }
-            // case 'UNKNOWN':
+            case 'UNKNOWN':
             default:
                 this.respond(421, 'Invalid PROXY format');
-                return this.disconnect();
+                return this.disconnect(); // Invalid address: return error function
```

Comment:
This is the head of Duplicate Redundant Department calling. The words 'return',  'break', and 'valid' in these comments seems a tad redundant, eh? What additional information and/or clarity is being added?  I'd argue that the comments are actually misleading. What "loop" is being broken? Is an invalid address the only condition that would hit the default case?

Reply:
Fixing now...